packages/@webex/plugin-meetings/src/meeting-info/utilv2.ts

```diff
@@ -191,26 +192,17 @@ MeetingInfoUtil.getDestinationType = async (from) => {
       return Promise.resolve(options);
     });
   } else if (hydraId.room) {
-    options.type = _CONVERSATION_URL_;
-    try {
-      await webex.internal.services.waitForCatalog('postauth');
-
-      const conversationUrl = webex.internal.conversation.getUrlFromClusterId({
-        cluster: hydraId.cluster,
-        id: hydraId.destination,
-      });
-
-      options.destination = conversationUrl;
-    } catch (e) {
-      LoggerProxy.logger.error(`Meeting-info:util#getDestinationType --> ${e}`);
-      throw e;
-    }
+    LoggerProxy.logger.error(
+      `Meeting-info:util#getDestinationType --> Using the space ID as a destination is no longer supported. Please refer to the [migration guide](https://github.com/webex/webex-js-sdk/wiki/Migration-guide-for-USM-meeting) to migrate to use the meeting ID or SIP address.`
+    );
+    // Error code 101 added as Space ID deprecated as of beta, Please refer migration guide.
+    throw new SpaceIDDeprecatedError();
   } else {
     LoggerProxy.logger.warn(
-      "Meeting-info:util#getDestinationType --> ('MeetingInfo is fetched with meeting link, sip uri, phone number, hydra room id, hydra people id, or a conversation url."
+      "Meeting-info:util#getDestinationType --> ('MeetingInfo is fetched with the meeting link, SIP URI, phone number, Hydra people ID, or a conversation URL."
     );
     throw new ParameterError(
-      'MeetingInfo is fetched with meeting link, sip uri, phone number, hydra room id, hydra people id, or a conversation url.'
+      'MeetingInfo is fetched with the meeting link, SIP URI, phone number, Hydra people ID, or a conversation URL.'
```

Comment:
Can we create a string for these two duplicating sentences so that we need not make change in two different places?

Reply:
@mkesavan13, I will check it and update.

Reply:
@mkesavan13, Addressed, check once..